qemu/tests/cfg/nested_test.cfg

```diff
@@ -11,10 +11,12 @@
     type = nested_test
     test_type = testcase
     install_node = no
-    test_timeout = 3600
+    login_timeout = 360
     variants nested_test:
         - boot_l2:
             case_name = boot
             install_node = yes
+            test_timeout = 5400
         - check_cpu_model_l2:
             case_name = x86_cpu_model
+            test_timeout = 3600
```

Comment:
`install_node = no` for this case, So i think 3600s is too long for `x86_cpu_model`. How do you think about 30mins - 1800s ?